nomad/fsm.go

```diff
@@ -51,6 +51,7 @@ const (
 	ScalingPolicySnapshot
 	CSIPluginSnapshot
 	CSIVolumeSnapshot
+	ScalingEventsSnapshot
```

Comment:
smol smol ting, just double checking this snapshot case denotes multiple scaling events?

Reply:
these snapshots are a single entry in the table. in this case, all of the scaling events for a single job.